### ISSUE TITLE
LOA1 gate page - incorrect accessibility hint

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#22122]
## What happened?
On the LOA1 gate page, the screen reader announces the **Read more if you haven't verified** button as:
"Read more if you haven't yet verified. Spin button. Collapsed. _Appeal details. View past events at eleven-y hint._"

The italicized text above is not the correct accessibility hint.

## Specs:
- Device: iOS iPhone 11 (VoiceOver), Samsung Galaxy S9 (Voice Assistant)

## Steps to Reproduce
- Navigate to the LOA 1 gate page
- Turn on VoiceOver or Voice Assistant
- Select the **Read more if you haven't verified** button
- Observe announcement

## Desired behavior
The accessibilityHint should be announced as: "view what you will need to verify your identity"

## Acceptance Criteria
- See above.

## Bug Severity
- 4 - Low

## Linked to story
#15100

## Screen shot(s) and additional information
![Image from iOS (30).png](https://images.zenhubusercontent.com/5f64e41291069a617acc1b47/3c503064-aea9-4fc3-a9a8-c023133c9ae9)

**Comment by digitalken** [comment]
PASS 3/29/2021 - Sounds good on Android and iOS.  Closing.
